app/assets/javascripts/listings/components/browseListings.js.coffee

```diff
@@ -37,7 +37,7 @@ angular.module('dahlia.components')
         children = $translate.instant(if @eligibilityFilters.children_under_6 == '1' then 't.child' else 't.children')
         header.push($translate.instant('listings.including_children', {number: @eligibilityFilters.children_under_6, children: children}))
 
-      timeframe = $translate.instant('t.' + $filter('incomeTimeframe')(@eligibilityFilters.income_timeframe))
+      timeframe = $translate.instant('t.per_' + $filter('incomeTimeframe')(@eligibilityFilters.income_timeframe))
```

Comment:
This one is going to be hard for the grunt translations script to pick up and detect. It's slightly less clean but it'd be easier to maintain if you explicitly say `$translate.instant('t.per_month')` or `$translate.instant('t.per_year')`